optuna/structs.py

```diff
@@ -277,7 +292,7 @@ class StudySummary(object):
     Attributes:
         study_name:
             Name of the :class:`~optuna.study.Study`.
-        direction:
+        # direction:
```

Comment:
It looks that this line is not relevant to this pull request.

```suggestion
        direction:
```

Reply:
Thank you. Indeed, it's an unintended change. I'll fix it.